src/controllers/movie.js

```diff
@@ -0,0 +1,204 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+const getMovies = async (req, res) => {
+  let movies
+
+  if (req.query.runtimeGt || req.query.runtimeLt) {
+    const runtimeLt = Number(req.query.runtimeLt)
+    const runtimeGt = Number(req.query.runtimeGt)
+
+    console.log(runtimeLt, runtimeGt)
+
+    if (req.query.runtimeGt && req.query.runtimeLt) {
+      // console.log('in both query')
+      movies = await prisma.movie.findMany({
+        where: {
+          runtimeMins: {
+            lt: runtimeLt,
+            gt: runtimeGt
+          }
+        },
+        include: {
+          screenings: true
+        }
+      })
+    }
+    else if (req.query.runtimeLt) {
```

Comment:
Nice work on the logic. The next step here would be to figure out a way to get rid of using a big else/if chain. One such solution could be something like:

```js
const { runtimeLt, runtimeGt } = req.query
const query = {}

// If either runtimeLt or runtimeGt are provided, add a `where` to the query
// If both are provided, add an AND to the where

const movies = await prisma.movie.findMany(query)
```

Switching your focus a little for this function will help. E.g. You have `prisma.movie.findMany` in here multiple times. How do you get to a point where you don't have to repeat the same code? (The answer: build the query first, then you can reduce the findMany to a single line)